src/Modules/Profile/Commands.Profile/InvokePowerBIRestMethod.cs

```diff
@@ -45,12 +47,22 @@ public class InvokePowerBIRestMethod : PowerBICmdlet
         public string Version = "v1.0";
 
         [Parameter(Mandatory = false)]
-
         public virtual string OutFile { get; set; }
+
+        [Parameter(Mandatory = false)]
+        public string ContentType { get; set; }
+
+        [Parameter(Mandatory = false)]
+        public Hashtable Headers { get; set; }
```

Comment:
Can you make this IDictionary?

Reply:
No PowerShell doesn't operate well with dictionaries, it operates with Hashtables instead. You would have to use New-Object instead of @{ … } syntax